cmd/exit.go

```diff
@@ -14,10 +14,14 @@ const (
 	// 5: CodeNotFound
 	CodeFailedDetect           = 100
 	CodeFailedDetectWithErrors = 101
-	CodeFailedBuild            = 7
+	CodeDetectError            = 102
```

Comment:
@ekcasey I wasn't sure if we wanted to document this in the spec explicitly or not. If we did, I guess we would want to do that for the other generic phase failures.

Reply:
I think it is sufficiently documented to the extent that it is in the detect-specific lifecycle error range (102-199).